src/main/java/estaru/birthdaygreetings/Friend.java

```diff
@@ -0,0 +1,43 @@
+package estaru.birthdaygreetings;
+
+import java.time.LocalDate;
+
+public class Friend {
+    private String firstName;
+    private String lastName;
+    private LocalDate dateOfBirth;
+
+    private String email;
+
+    public Friend(String firstName, String lastName, LocalDate dateOfBirth, String email) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.dateOfBirth = dateOfBirth;
+        this.email = email;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
```

Comment:
that method should return lastName not firstName. I wonder why it didn't show up in the test